examples/native-hybrid-app/README.md

```diff
@@ -10,7 +10,11 @@ If you are using an applicaiton that is already using the native Datadog iOS or
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
-  await DatadogSdk.instance.attachToExisting();
+  final config = DdSdkExistingConfiguration(
+    loggingConfiguration: LoggingConfiguration()
```

Comment:
Can we add the other attaching parameters to the README as well?

Reply:
I added a note in the README to check the pub.dev documentation for a full list.